__tests__/pages/my-dashboard.test.js

```diff
@@ -27,11 +27,32 @@ jest.mock('../../components/Card', () => {
   return MockCard
 })
 
+jest.mock('../../components/ContextualAlert', () => {
+  const MockAlert = () => <mock-alert data-testid="mock-alert" />
+  return MockAlert
+})
 describe('My Dashboard page', () => {
   const content = {
     heading: 'heading',
     paragraph: 'paragraph',
-    cards: [{ id: 'test', title: 'title', lists: [] }],
+    cards: [
+      {
+        id: 'test',
+        title: 'title',
+        cardAlerts: [
+          {
+            id: 'test',
+            type: 'title',
+            alertHeading: 'heading',
+            alertBody: 'body',
+          },
+        ],
+        lists: [],
+      },
+    ],
+    pageAlerts: [
+      { id: 'test', type: 'title', alertHeading: 'heading', alertBody: 'body' },
```

Comment:
I know this is just a mocked test but instead of having type set to `title`, we should set it to a somewhat realistic value (ie. info, danger, etc)

Reply:
fixed